console/atest-desktop/main.js

```diff
@@ -108,12 +110,25 @@ app.whenReady().then(() => {
     log.info('start to write file with length %d', data.length)
     
     try { 
-      fs.writeFileSync(atestFromHome, data);
-    } 
+	if (process.platform === "win32") {
+		const file = fs.openSync(atestFromHome, 'w');
+		fs.writeSync(file, data, 0, data.length, 0);
+		fs.closeSync(file);
+	}else{
+		fs.writeFileSync(atestFromHome, data);
+	}
+	} 
     catch (e) { 
       log.error('Error Code: %s', e.code); 
+	  log.error('Error writing atest.exe to %s: %s', atestFromHome, e.message);
     }
-  }
+	if (fs.existsSync(atestFromHome)) {
+		log.info('atest.exe file exists at ${atestFromHome}')
+	} else {
+		log.error('Failed to write atest.exe to ${atestFromHome}')
+	}
+	}
+  
```

Comment:
Forget to format the codes?

Reply:
I'm sorry, this was my test and I forgot to delete it.

Reply:
By the way, please don't forget to sign CLA.

![image](https://github.com/LinuxSuRen/api-testing/assets/1450685/616e704e-d010-4145-bb83-3bb3114d7a82)